app/.eslintrc.js

```diff
@@ -3,11 +3,12 @@ module.exports = {
   extends: '@react-native',
   plugins: ['eslint-plugin-no-inline-styles'],
   rules: {
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/no-shadow': 'off',
```

Comment:
why we want to disable it here?

Reply:
+1

Reply:
To limit number of changes inside one PR

Reply:
I don't think we really care about number of changes in `/app`.